docker/py39/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.9
+
+RUN apt-get update \
+    && apt-get install -y \
+        hdf5-tools \
+        curl
```

Comment:
Should add ` && rm -rf /var/lib/apt/lists/*` according to best practices:
https://docs.docker.com/develop/develop-images/instructions/

Reply:
Will do. Thanks for the link.